package.json

```diff
@@ -7,8 +7,9 @@
     "node": ">=5"
   },
   "scripts": {
+    "closure": "closure/closure-type-checking.js",
     "lint": "eslint .",
-    "test": "npm run lint --silent && npm run unit",
+    "test": "npm run lint --silent && npm run closure && npm run unit",
```

Comment:
pretty sure closure is going to be the slowest of these. i'd be interested in sorting these fast to slow.

    npm run unit && npm run lint --silent && npm run closure